src/java_tools/import_deps_checker/java/com/google/devtools/build/importdeps/DepsCheckerClassVisitor.java

```diff
@@ -58,7 +58,9 @@ public void visit(
       String[] interfaces) {
     checkState(internalName == null, "Cannot reuse this class visitor %s", getClass());
     this.internalName = name;
-    checkInternalName(superName);
+    if (superName != null) {
+      checkInternalName(superName);
```

Comment:
please add a comment stating when this will happen, i.e., something like:

// module-info and java.lang.Object have null superName